src/extensions/default/bramble/lib/ConsoleManager.js

```diff
@@ -0,0 +1,29 @@
+define(function (require, exports, module) {
+    "use strict";
+
+    var BrambleEvents = brackets.getModule("bramble/BrambleEvents");
+    var EditorManager = brackets.getModule("editor/EditorManager");
+    var CommandManager = brackets.getModule("command/CommandManager");
+    var Commands = brackets.getModule("command/Commands");
+
+    var ConsoleManagerRemote = require("text!lib/ConsoleManagerRemote.js");
+    var IFrameBrowser = require("lib/iframe-browser");
+
+    var isConsoleEnabled = false;
+
+    function getRemoteScript() {
+        return "<script>\n" + ConsoleManagerRemote + "</script>\n";
```

Comment:
Yes, the idea I had was not to load it "as text", but instead make it load the usual way, so that you end up generating a `<script src="lib/ConsoleManagerRemote.js"></script>` in some way. After all, if the code can run "as string", it should be able to run as a normal script `src`, too. The difference being that when you load it as resource from a URL, that drives down the payload size because the browser can cache the file and doesn't need to redownload it every time, cutting down on load time.